src/braket/default_simulator/cirq/program_context.py

```diff
@@ -0,0 +1,88 @@
+from typing import Tuple, List
+
+import cirq
+import numpy as np
+from braket.ir.jaqcd.program_v1 import Results
+from cirq import Circuit
+
+from braket.default_simulator.openqasm.program_context import AbstractProgramContext
+
+CIRQ_GATES = {
+    "i": cirq.I,
+    "h": cirq.H,
+    "x": cirq.X,
+    "y": cirq.Y,
+    "z": cirq.Z,
+    "cnot": cirq.CNOT,
+    "cz": cirq.CZ,
+    "s": cirq.S,
+    "t": cirq.T,
+    "cphaseshift": cirq.cphase,
+    "rx": cirq.rx,
+    "ry": cirq.ry,
+    "rz": cirq.rz,
+    "swap": cirq.SWAP,
+    "iswap": cirq.ISWAP,
+    # "xx": cirq.XX,
+    # "yy": cirq.YY,
+    # "zz": cirq.ZZ,
```

Comment:
I think that should still be fine. Two gates (braket / cirq) with the same name should work on their respective simulators at the same time. Just here, no parameters will be expected.

However, it may be confusing to customers trying to use the xx gate from the BDK, since that refers to the Braket xx gate specifically. I'm ok with leaving these out for now

Reply:
Since we are focusing to convert a braket specific openqasm strings. 
Even if I add a conditional check for these gates to make an exception to construct the circuit it would not be equivalent to the input braket circuit.

Do I just comment them or remove them all together similar to other gates which I was not able to find in the documentation.

Reply:
I would remove them and add a comment above `CIRQ_GATES` noting that xx, yy, and zz are not the same gates between Braket and Cirq